### PR TITLE
Formatting

### PR DESCRIPTION
**Pull request by knatten**
This formats all the code with `autopep8`, adds a script to do that, and enforces formatting in CI. Thanks to @tocic for the idea in #270.

### CODE REVIEW

File: ci/format.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+cd $ROOT_DIR || exit $?
+
+EXCLUDES=$(tr '\n' ',' < .gitignore)
+EXCLUDES="$EXCLUDES,.*/migrations"
+IGNORES="E402" # module level import not at top of file (breaks how settings include local settings)
+
+if [ "$1" == "--fix" ]; then
+    autopep8 --ignore=$IGNORES --in-place --recursive --exclude $EXCLUDES . || exit $?
+fi
```
Comment (tocic) [inline ci/format.sh:14]: Perhaps it's easier to write a single configuration file than to construct command-line arguments.
See https://pypi.org/project/autopep8/#configuration.

**Review by tocic — APPROVED** [review]

File: ci/format.sh (hunk 1 above)
Comment (knatten) [reply ci/format.sh]: Good idea, fixed that now. Also put a bit more thought into this, extended the max line length and made some manual (but stable) tweaks. Pretty happy with this PR now! Thanks a lot for pushing to get this done.

Comment (knatten) [reply ci/format.sh]: I can btw wait with merging this until I've had time to review and merge your other PR, then you don't have to rebase on formatting changes.

Comment (tocic) [reply ci/format.sh]: No need to block the PR, merge as you wish. I'll rebase if necessary.

Comment (knatten) [reply ci/format.sh]: Cool, thanks a lot! I'll merge now then.

**Pull request merged**